content/6_Advanced/Game_Theory.mdx

```diff
@@ -2,35 +2,67 @@
 id: game-theory
 title: 'Game Theory'
 author: Benjamin Qi
-description: '?'
+contributors: Salma J
+description: 'Problems involving Algorithmic Game Theory'
 frequency: 0
 ---
 
-<IncompleteSection>
-
-make separate module for games w/o nimbers?
-
-</IncompleteSection>
-
-- Tutorial
-  - [HackerRank](https://www.hackerrank.com/topics/game-theory-and-grundy-numbers)
-  - [TopCoder](https://www.topcoder.com/community/data-science/data-science-tutorials/algorithm-games/)
-  - Books (thanks mnbvmar!)
-    - "On Numbers and Games" - Conway
-    - "Winning Ways for Your Mathematical Plays" (Berlekamp, Conway, Guy)
-- Topics
-  - Nim
-  - Grundy Numbers
-  - Sprague-Grundy Theorem
-- Problem(s)
-  - [Division Game](https://onlinejudge.org/index.php?option=onlinejudge&page=show_problem&problem=2959)
-  - [Game of Stones](http://codeforces.com/problemset/problem/768/E)
-  - [Factorization Game](https://www.hackerearth.com/problem/algorithm/mancunian-and-factorization-game-b8794702/)
-  - [Arpa & Game](http://codeforces.com/contest/850/problem/C)
-  - [Apple Tree](http://codeforces.com/contest/812/problem/E)
+<IncompleteSection />
+
+<Resources>
+	<Resource
+		source="HackerRank"
+		title="Game Theory and Grundy Numbers"
+		url="https://www.youtube.com/watch?v=JXTVOyQpSGM"
```

Comment:
wrong url

Reply:
i didnt put that lmao it was just there, I'll try finding the og link